translations/open-status.pot

```diff
@@ -0,0 +1,23 @@
+msgid ""
+msgstr ""
+"Content-Type: text/plain; charset=UTF-8\n"
+
+msgctxt "Whether a store is open or closed"
+msgid "Open 24 Hours"
+msgstr ""
+
+msgctxt "Whether a store is open or closed"
+msgid "Closed"
+msgstr ""
+
+msgctxt "Whether a store is open or closed"
```

Comment:
nit: This context might be a little confusing for this particular translation. How about something like: "Indicates the time that a store will open"

Reply:
yeah makes sense